index.html

```diff
@@ -84,7 +107,7 @@ <h3 class="text_titleH3">Project 2</h3>
             <div class="container-work" id="project3">
                 <div class="img-work img-work-source3"></div>
                 <div class="div-title flex-work-title">
-                    <h3 class="text_titleH3">Project 3</h3>
+                    <h3 class="text-titleH3">Project 3</h3>
                 </div>
                 <div class="flex-ul-work">
                     <div class="ul-work-item">
```

Comment:
Not to be repetitive, but kindly make sure this project is closer to 50% of the screen, that way, your project will look closer to the Figma design 😅

Reply:
Hello @Kakalanp,

I'm confusing, could you explain what is the problem with the class text-titleH3?

Reply:
I'm sorry I didn't attach an image to guide you, I was talking about the project card in general, but you nailed it! :ok_hand: